pkgs/development/python-modules/biliass/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, protobuf
+}:
+
+buildPythonPackage rec {
+  pname = "biliass";
+  version = "1.3.4";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-kktK+6rLwYhkG7LiTBlgBbiIN8apweg4l8pJSTjKQU4=";
+  };
+
+  propagatedBuildInputs = [ protobuf ];
+
+  pythonImportsCheck = [ pname ];
```

Comment:
You should type the name out instead of using `pname`, for readability's sake

Reply:
Done.